MaterialSkin/Plugin.pm

```diff
@@ -830,7 +833,7 @@ sub _cliCommand {
             $params->{'saveSearch'} = $saveLib;
         }
 
-        my ($tracks, $albums) = Plugins::MaterialSkin::Search::advancedSearch($params);
+        my ($tracks, $albums) = Plugins::MaterialSkin::Search::advancedSearch($request->client(), $params);
```

Comment:
Why is a client required? Material does not passa client into this API, as it is not used.